src/presets.asm

```diff
@@ -549,7 +550,7 @@ add_grapple_and_xray_to_hud:
 }
 
 print pc, " preset_start_gameplay end"
-warnpc $80FC00
+warnpc $80F800 ; save.asm
```

Comment:
Disappointed I missed this, good catch

Reply:
I have some extra stuff in there for randomizing equipment, which comes with $160 bytes of tables. I also moved the LoadRandomPreset routine there since I ran out of menu space and had to cut/move some stuff out of $B8. So I didn't catch it, someone getting a savestate build did :)